run

```diff
@@ -856,8 +856,8 @@ for k, v in obj[project_type]["run"].items():
       workdir="cd ${holohub_build_dir}"
    fi
 
-   local environment="export PYTHONPATH=\${PYTHONPATH}:${holoscan_sdk_install}/../../../python/lib:${holohub_build_dir}/python/lib:${SCRIPT_DIR} && export HOLOHUB_DATA_PATH=${holohub_data_dir}"
-   local reset_environment="export PYTHONPATH=${PYTHONPATH} && export HOLOHUB_DATA_PATH=\"${HOLOHUB_DATA_PATH}\""
+   local environment="export PYTHONPATH=\${PYTHONPATH}:${holoscan_sdk_install}/../../../python/lib:${holohub_build_dir}/python/lib:${SCRIPT_DIR} && export HOLOHUB_DATA_PATH=${holohub_data_dir} && export HOLOSCAN_INPUT_PATH=${holohub_data_dir}"
```

Comment:
Is this required by the CLI packager?

Reply:
The `HOLOSCAN_INPUT_PATH` environment variable is required by [HAP](https://docs.nvidia.com/holoscan/sdk-user-guide/cli/hap.html) and the CLI.

Reply:
Got it, thanks for the link.